common/src/integration-test/kotlin/org/web3j/corda/network/CordaNode.kt

```diff
@@ -165,7 +170,8 @@ abstract class CordaNode internal constructor(protected val network: CordaNetwor
     }
 
     companion object : KLogging() {
-        private const val CORDA_ZULU_IMAGE = "corda/corda-zulu-4.1:latest"
+        private const val CORDA_ZULU_VERSION = "latest"
+        private const val CORDA_ZULU_IMAGE = "corda/corda-zulu-4.1"
```

Comment:
I think we should be able to customize the name as well. https://hub.docker.com/r/corda/corda-zulu-4.3-rc01 looks like the latest version, but that means we should have the ability to change name of the corda docker image as well.

Reply:
@Puneetha17 would you reckon to allow specifying the whole name eg `corda/corda-zulu-4.3-rc01`or just the image name `corda-zulu-4.3-rc01`?

Reply:
Also, this would apply only for the COrda node but for the network map I think specifying the tag would be enough right?